apps/src/templates/TeacherFeedback.jsx

```diff
@@ -22,12 +23,23 @@ const styles = {
   },
   content: {
     padding: 10
+  },
+  textInput: {
+    margin: 10,
+    display: 'block',
+    width: '100%',
+  },
+  button: {
+    margin: 10,
+    fontWeight: 'bold'
   }
 };
 
 class TeacherFeedback extends Component {
   static propTypes = {
-    viewAs: PropTypes.oneOf(['Teacher', 'Student'])
+    viewAs: PropTypes.oneOf(['Teacher', 'Student']),
```

Comment:
There a `const` for that:

https://github.com/code-dot-org/code-dot-org/blob/6ddaece27a8d2857e6f101b7daba6697957e1a1f/apps/src/templates/progress/SummaryProgressRow.jsx#L87